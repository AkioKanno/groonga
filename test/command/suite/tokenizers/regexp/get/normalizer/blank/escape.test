table_create Lexicon TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenRegexp \
  --normalizer NormalizerAuto
table_tokenize Lexicon "abc\ndef" --mode ADD

table_tokenize Lexicon "a\\bc\ndef" --mode GET
