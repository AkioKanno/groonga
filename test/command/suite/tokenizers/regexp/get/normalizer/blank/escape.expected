table_create Lexicon TABLE_PAT_KEY ShortText   --default_tokenizer TokenRegexp   --normalizer NormalizerAuto
[[0,0.0,0.0],true]
table_tokenize Lexicon "abc\ndef" --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "￯",
      "position": 0
    },
    {
      "value": "ab",
      "position": 1
    },
    {
      "value": "bc",
      "position": 2
    },
    {
      "value": "c",
      "position": 3
    },
    {
      "value": "de",
      "position": 5
    },
    {
      "value": "ef",
      "position": 6
    },
    {
      "value": "f",
      "position": 7
    },
    {
      "value": "￰",
      "position": 8
    }
  ]
]
table_tokenize Lexicon "a\\bc\ndef" --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "ab",
      "position": 0
    },
    {
      "value": "bc",
      "position": 1
    },
    {
      "value": "de",
      "position": 3
    },
    {
      "value": "ef",
      "position": 4
    }
  ]
]
